.github/workflows/publish_unit_tests_job.yml

```diff
@@ -42,5 +42,5 @@ jobs:
         uses: lucassabreu/comment-coverage-clover@main
         with:
           file: "/home/runner/work/opg-pdf-service/opg-pdf-service/coverage/clover.xml"
-          min-line-coverage: 89
-          min-method-coverage: 100
+          #min-line-coverage: 89
+          #min-method-coverage: 100
```

Comment:
Should probably keep this unless we have good reason not to. If we do need to remove it, we should delete the whole step since it won't really do anything without these set